src/charm.py

```diff
@@ -9,6 +9,7 @@
 
 import yaml
 from charmed_kubeflow_chisme.exceptions import ErrorWithStatus
+from charms.data_platform_libs.v0.data_interfaces import DatabaseRequires
```

Comment:
Is this database interface a required interface?

Reply:
Its used to create the database and register new events. So I think it is

Reply:
yes, required

Reply:
We would need to block charm if relation does not exist, right?

Reply:
yes we do it ... mlflow cant start without relational db